velox/vector/VectorSaver.h

```diff
@@ -22,6 +22,9 @@ namespace facebook::velox {
 // Serialization format used by the following function is documented at
 // https://facebookincubator.github.io/velox/develop/debugging/vector-saver.html
 
+static constexpr folly::StringPiece kVectorPathPrefix = "vector";
```

Comment:
I'm not sure it makes sense to have these constants. The callers should decide for themselves how to name the files. In particular, I don't think there is a use case for kStringPathPrefix.